templates/index.html

```diff
@@ -1,7 +1 @@
-<title>Community website</title>
-<body>
-<ul>
-    <li><a href="/gci/">Google Code-in</a>
-    <li><a href="/activity/">GitHub activity</a>
-</ul>
-</body>
+{% include 'activity.html' %}
```

Comment:
This is the front page of the website.  Please enhance it.   `activity.html` could be improved , with header and footer, but its content shouldnt be modified.